src/ott/tools/plot.py

```diff
@@ -424,3 +439,513 @@ def animate(
         interval=1000.0 / frame_rate,
         blit=True,
     )
+
+
+def get_plotkwargs(
+    background: bool,
+    small_alpha: float = 0.2,
+    large_alpha: float = 0.7,
+    darkmode: bool = False,
+    small_size: int = 50,
+    mid_size: int = 60,
+    size_multiplier: float = 1.2
+) -> dict:
+  r"""Generate marker styling specifications for transport visualization.
+
+  This utility function creates a dictionary of matplotlib styling parameters
+  for various types of points and arrows used in optimal transport
+  visualizations. It provides consistent color schemes for both light and dark
+  modes and handles different marker types for source, target, and intermediate
+  points.
+
+  Args:
+    background: Whether points in source (``x``) and target (``y``) sets should
+      have small alphas to de-emphasize them and highlight other elements like
+      dynamic points or arrows.
+    small_alpha: Alpha (transparency) value for background points.
+      Default is ``0.2``.
```

Comment:
Please don't mention defaults (they are already in the function signature); also elsewhere in this file.